etc/kernel-launchers/kubernetes/scripts/kernel-pod.yaml.j2

```diff
@@ -54,6 +54,11 @@ spec:
       value: "{{ kernel_namespace }}"
     image: "{{ kernel_image }}"
     name: "{{ kernel_pod_name }}"
+    # resources:
+    #   limits:
+    #     cpu: "1"
+    #     memory: "2Gi"
+    #     nvidia.com/gpu: "0"
```

Comment:
Are the `requests` stanzas similar?  If so, I think it would make sense to add a `requests` stanza, still commented out, but rather than using fixed values, use the following parameters, `{{ kernel_cpus }}`, `{{ kernel_memory }}`, and `{{ kernel_gpus }}`.  This would tie in nicely for any parameterization efforts moving forward.

Reply:
Yep `requests` are similar, and it would be better to not use fixed values indeed.